shell/browser/api/electron_api_url_loader.cc

```diff
@@ -381,6 +381,11 @@ gin::Handle<SimpleURLLoaderWrapper> SimpleURLLoaderWrapper::Create(
   if (!origin.empty()) {
     request->request_initiator = url::Origin::Create(GURL(origin));
   }
+  bool has_user_activation;
+  if (opts.Get("hasUserActivation", &has_user_activation)) {
+    request->trusted_params = network::ResourceRequest::TrustedParams();
+    request->trusted_params->has_user_activation = has_user_activation;
```

Comment:
It looks like we'll start sending `Sec-Fetch-User: ?0` with all requests by default as of this change. Is that desirable?

Reply:
I don’t think so. But I will add a test to ensure this.

Reply:
It seems the chromimum network stack will remove the Sec-Fetch-User header if has_user_activation is false.

Reply:
I added a test to make sure this doesn't happen.